### PR TITLE
Add socials page

### PR DESCRIPTION
**Pull request by alexcostaluiz**
Add a "socials" page with links to various professional social network profiles.

### CODE REVIEW

File: portfolio/src/main/webapp/social.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	          </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+
+    <script>
+      var socials = document.querySelector(".socials")
+      Animator.queue(new FadeShift(socials, 1, 0.1, 'y', 24, 0, -1.6))
+    </script>
```
Comment (jesseelizabeth) [inline portfolio/src/main/webapp/social.html:76]: This is such a small script, so I understand wanting to put it in the html file. However it's best to separate concerns as much as possible (i.e. separate your HTML, from your CSS, from your JS). Just something to keep in mind :) https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

File: portfolio/src/main/webapp/social.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	              <td><a class="button" target="_blank" href="https://app.joinhandshake.com/users/14864749">app.joinhandshake.com/users/14864749</a></td>
+	            </tr>
+	          </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+
+    <script>
+      var socials = document.querySelector(".socials")
```
Comment (jesseelizabeth) [inline portfolio/src/main/webapp/social.html:74]: nit: always use const or let https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

File: portfolio/src/main/webapp/social.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  </head>
+  <body>
+
+    <div class="menu container u-fixed">
+      <div class="row">
+        <div class="column">
+          <a href="/">.home</a><br>
+          <a href="/about">.about</a><br>
+          <a href="/projects">.projects</a><br>
+          <!-- <a href="/essays">.essays</a><br> -->
```
Comment (jesseelizabeth) [inline portfolio/src/main/webapp/social.html:26]: Uncomment?

**Comment by alexcostaluiz** [comment]
Thanks for all the recommendations! I implemented all of them and decided to go with creating a separate js file for the on-load animations to completely separate my html from my js (instead of doing <body onload="function()"> since this onload listener waits for the entire page including images to be downloaded before running and I want the animation to run closer to onDOMContentLoaded). PR #20 is this new js file.

**Comment by jesseelizabeth** [comment]
Looks great! Event listeners can be super useful, good to see you learning how they work and understanding the benefits they provide. Thanks for making these changes!

**Pull request merged**